websites/T/Timeguessr/presence.ts

```diff
@@ -0,0 +1,81 @@
+const presence = new Presence({
+		clientId: "1327710845546922075",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/AdqBPUA.jpeg",
+			startTimestamp: browsingTimestamp,
+		},
+		privacy = await presence.getSetting<boolean>("privacy"),
+		{ pathname } = window.location;
```

Comment:
For consistency across activities
```suggestion
		{ pathname } = document.location;
```